pkg/steampipeconfig/options/database.go

```diff
@@ -9,6 +9,7 @@ import (
 
 // Database
 type Database struct {
+	Host         *string `hcl:"host"`
```

Comment:
Note: adding that option requires rebuilding `steampipe-postgres-fdw` with its `github.com/turbot/steampipe` dependency updated to this branch as it loads configuration files too:
* https://github.com/turbot/steampipe-postgres-fdw/blob/ba544f620e99d4fa6b1379fbfe59bfda0d512150/fdw.go#L76
* https://github.com/turbot/steampipe-postgres-fdw/blob/ba544f620e99d4fa6b1379fbfe59bfda0d512150/hub/hub.go#L319

Reply:
Removed.